include/nlohmann/detail/output/output_adapters.hpp

```diff
@@ -97,7 +97,6 @@ class output_string_adapter : public output_adapter_protocol<CharType>
   private:
     StringType& str;
 };
-
```

Comment:
Sorry to be pedantic, but can you please re-add the line?

Reply:
Oh, sorry, will do now.